bin/i18n/test/translation_status/test_translation_service.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../../../../shared/test/common_test_helper'
+require_relative '../../translation_status/translation_service'
+
+class TranslationServiceTest < Minitest::Test
+  # Use the same service for all the tests because intialization is very slow
+  @@translation_service = TranslationService.new
+
+  def test_translated_given_english_hello_should_return_true
+    assert @@translation_service.translated?(:en, 'hello')
+  end
+
+  def test_translated_given_german_hello_should_return_true
+    assert @@translation_service.translated?(:de, 'hello')
```

Comment:
Question: is the assumption here that 'hello' is so common, we must already have a translation for it in all environments, including dev, staging and test?

Reply:
dev, staging, and test all use the same translation files, so yes. This is a "Hello, world" kind of string that has been around for a long time.